tests/app/views/handlers/test_confirmation_email_fulfilment_request.py

```diff
@@ -26,7 +26,7 @@ def session_data():
         survey_url=None,
         ru_name=None,
         ru_ref=None,
-        submitted_time=datetime.now(tzutc()).isoformat(),
+        submitted_at=datetime.now(tzutc()).isoformat(),
```

Comment:
The code used to get the current time should be consistent throughout i.e. `datetime.now(timezone.utc)`.